src/api/config/environment.rb

```diff
@@ -18,13 +18,5 @@
 CONFIG['proxy_auth_mode'] ||= :off
 CONFIG['frontend_ldap_mode'] ||= :off
 
-# use hardcoded source server port for test and development
```

Comment:
they are already defined in config/environments/test.rb and developent.rb. I have just removed the double definition.